Microsoft.Alm.Authentication/Secret.cs

```diff
@@ -19,7 +19,7 @@ public static string UriToSimpleName(TargetUri targetUri, string @namespace)
             string trimmedHostUrl = targetUri.Host
                                              .TrimEnd('/', '\\')
                                              .TrimEnd();
-            Uri resolvedUri = targetUri.ResolvedUri;
```

Comment:
Is switching everything from `ResolvedUri` to `ActualUri` the right thing to do in every case?